onyo/lib/commands.py

```diff
@@ -89,49 +89,43 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
 
 def fsck(repo: OnyoRepo,
          tests: list[str] | None = None) -> None:
-    r"""Run a suite of integrity checks on an Onyo repository and its contents.
+    r"""Run integrity checks on an Onyo repository and its contents.
 
-    By default, the following tests are performed:
+    The following tests are available:
 
-    * ``anchors``: verify that all directories (outside of ``.onyo/``) have an
-      ``.anchor`` file
-    * ``asset-unique``: verify that all asset names are unique
-    * ``asset-yaml``: verify that all asset contents are valid YAML
-    * ``clean-tree``: verify that git has no changed (staged or unstaged) or
-      untracked files
+    * ``anchors``: directories (outside of ``.onyo/``) have an ``.anchor`` file
```

Comment:
I think somewhere here `.gitignore` needs to be mentioned.

Reply:
Done.